### PR TITLE
Add a log handler

### PR DESCRIPTION
**Pull request by jderusse**
Provide a simple way to configure Monolog's NewRelicHandler

```yaml
ekino_new_relic:
  log_logs: ~
```

or a more advanced usage
```yaml
ekino_new_relic:
  log_logs:
    level: warning
    channels: [app, event]
    service: ekino.new_relic.logs_handler.auto
```

TODO:

* [x] Missing tests

### CODE REVIEW

File: Logging/AutoHandler.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    ) {
+        parent::__construct($level, $bubble, $appName, $explodeArrays, $transactionName);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function write(array $record)
+    {
+        if (!$this->isNewRelicEnabled()) {
```
Comment (Nyholm) [inline Logging/AutoHandler.php]: You use this to avoid throwing an exception, right?

File: DependencyInjection/EkinoNewRelicExtension.php [hunk 2]
```diff
@@ -71,6 +72,20 @@ public function load(array $configs, ContainerBuilder $container)
...
+        if ($config['enabled'] && $config['log_logs']['enabled']) {
+            if (!class_exists(NewRelicHandler::class)) {
+                throw new \LogicException('The "monolog/monolog" package must be installed in order to use "log_logs" option.');
+            }
+            $container->getDefinition($config['log_logs']['service'])
+                ->replaceArgument(0, $this->levelToMonologConst($config['log_logs']['level']))
+                ->replaceArgument(2, $config['application_name']);
+
+            foreach ($config['log_logs']['channels'] as $channel) {
+                $def = $container->getDefinition($channel === 'app' ? 'monolog.logger' : 'monolog.logger.'.$channel);
```
Comment (Nyholm) [inline DependencyInjection/EkinoNewRelicExtension.php]: Could you really modify other bundles services like this?

Comment (jderusse) [reply DependencyInjection/EkinoNewRelicExtension.php]: no yhou're right, I can only if the bundle is initialized in the right order. I've to write a compiler pass

**Comment by jderusse** [comment]
Reday for review

**Comment by Nyholm** [comment]
Instead of using a compiler pass, could we tag our service so it automatically got picked up my MonologBundle?

**Comment by jderusse** [comment]
As far as I know, there is no tag in `symfony/monolog-bundle` providing such feature.

If you are thinking about `monolog.processor` the purpose of this tag is to inject the right channel's logger into the application.

see `Symfony\Bundle\MonologBundle\DependencyInjection\Compiler\DebugHandlerPass`

**Pull request merged**